Yoast/yoast-test-helper
# Implementing grunt

This PR contains:

- Grunt commands to create an Artifact folder (+zip)
- Grunt commands to version bump the plugin
- A new readme.txt, which is currently empty on purpose 
- The `grunt deploy:master` command
- The `grunt deploy:trunk` command
Test instructions:
Ensure you have all packages installed by running `yarn`.
Run the ` grunt artifact:zip` command, this should generate a new artifact zip.

Fixes #23
